src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -1021,18 +992,251 @@ define(function(require) {
 		 * @param  {jQuery} $template  Step template
 		 * @param  {Object} args  Wizard's arguments
 		 * @param  {Object} args.data  Wizard's data that is shared across steps
+		 * @param  {Object} eventArgs  Event arguments
+		 * @param  {Boolean} eventArgs.completeStep  Whether or not the current step will be
+		 *                                           completed
 		 * @returns  {Object}  Object that contains the updated step data, and if it is valid
 		 */
-		portWizardOwnershipConfirmationUtil: function($template, args) {
-			var self = this;
+		portWizardOwnershipConfirmationUtil: function($template, args, eventArgs) {
+			var self = this,
+				$form = $template.find('form'),
+				isValid = !eventArgs.completeStep || monster.ui.valid($form),
```

Comment:
Is this working well?

since `isValid` will get the value of `!eventArgs.completeStep`, due to `eventArgs.completeStep` will be defined with `true or false`, which means this `OR` condition `(|| monster.ui.valid($form))` isn't going to happen.

Does that make sense?

Reply:
Yes, it is working well. the form is validated (`monster.ui.valid($form)`) only when the step is going to be completed pressing the next button when it is the first time in the current step, or it is already completed and we are moving to any other step.